opencog/haskell/LICENSE

```diff
@@ -0,0 +1,693 @@
+Unless indicated otherwise (a license reference in file headers, or a 
+LICENSE file in a directory), all OpenCog source files use the AGPL 
+with additional permissions to link OpenCog libraries with non-AGPL
+works; the text of the OpenCog linking excpetion is located at the end
+of this file.
+
+ 
+
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Hi. Ok, I will remove it.
I added it because I tried to develop a complete cabal package that needs to include a license file. But as we are not planning to upload this package anywhere for now (hackage,etc) , I can remove the LICENSE file.